reference/6/Microsoft.PowerShell.Core/Enter-PSHostProcess.md

```diff
@@ -188,6 +195,24 @@ Accept pipeline input: True (ByValue)
 Accept wildcard characters: False
 ```
 
+### -CustomPipeName
+
+Gets or sets the custom named pipe name to connect to.
+This is usually used in conjunction with
+`pwsh -CustomPipeName`.
+
+```yaml
+Type: String
+Parameter Sets: PipeNameParameterSet
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
 ### CommonParameters
 
 This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
```

Comment:
Fix "about_CommonParameters" to be a valid hyperlink.

Reply:
Update ms.date to the current date in both articles

Reply:
fixed.